CHANGES.md

```diff
@@ -22,6 +22,7 @@ Release Notes.
   name doesn't exist in TCP traffic.
 * Upgrade H2 version to 2.0.202 to fix CVE-2021-23463.
 * Extend column name override mechanism working for `ValueColumnMetadata`.
+* Introduce new concept `Layer` and removed `NodeType`. More details refer to [v9-version-upgrade](docs/en/FAQ/v9-version-upgrade.md)
```

Comment:
This link has been updated after the releasing, as this file will be moved into `changes` folder.